packages/ethereum/cardstack/event-indexer.js

```diff
@@ -13,9 +13,8 @@ function attachMeta(model, meta) {
 
 module.exports = declareInjections({
   indexer: 'hub:indexers',
-  controllingBranch: 'hub:controlling-branch',
```

Comment:
@robustdj controllingBranch is no longer a thing. delete these (and downstream stuff) where every you see it.